src/components/PathwayCourse/index.js

```diff
@@ -341,12 +344,22 @@ function PathwayCourse() {
           open={openModal}
           aria-labelledby="modal-modal-title"
           aria-describedby="modal-modal-description"
-          onClose={handleModal}
+          onClose={onCloseHandle}
         >
           <Box sx={modalStyle}>
-            <Typography
-              sx={{ fontSize: "32px", fontWeight: "600" }}
-            >{`${pathwayCourseData?.pathway}  Certificate`}</Typography>
+            <div style={{ display: "flex", justifyContent: "space-between" }}>
+              <Typography
+                sx={{ fontSize: "32px", fontWeight: "600" }}
+              >{`${pathwayCourseData?.pathway}  Certificate`}</Typography>
+
+              <CloseIcon
+                sx={{ cursor: "pointer", marginTop: "15px" }}
+                onClick={() => {
+                  setOpenModal(false);
+                }}
+              />
+            </div>
+
             <div className={classes.pdfWrapper}>
               <iframe
                 allowtransparency="true"
```

Comment:
A lot of this is duplicated in `CertificateCard, so it should be separated into a new component. Remember we want to keep components as small as possible and each one should have a single purpose.

Reply:
I am not getting this point can you explain me? @jschanker

Reply:
@saloni-080601 Sure, ek component sirf ek baat karana chahiye (https://react.dev/learn/thinking-in-react dekho.) Phir component ek se jyaada jagah mein upayog kar sakata hai. Lekin yahan, `PathwayCourse` certificate ke design ke liye aur pathway ke courses dikhaane ke liye zimmedar hai. Yeh matalab karta hai ki hum `Profile` Component mein Certificate ka code upayog nahin kar sakte hai. Toh, hum `Profile` ke component mein kuch `PathwayCourse` ke code se dohara rahe hain. Agar hamare paas ek `Certificate` Component, toh hum donon `PathwayCourse` aur `Profile` ke component mein `Certificate` ka code upayog kar sakate. Phir, humein yeh code doharanaa nahin karnaa hai. Iske alaava, har component ka code aur bhi chhota hoga isliye code aur aassaani se samjhnaa hoga.

Agar meree hindee galat hai, toh mujhe jaane dijiye. Main seekhane ke koshish kar raha hoon.

Reply:
ok

Reply:
Well done explanation :)

Reply:
@komalahire Dhanyavad! Main bas abhee yeh message dekh raha hoon kyonki mere e-mail mein GitHub notifications praapt nahin ho rahe the. Mujhe Hindi mein aur practice karna chahiye.